docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,127 @@
-
-
-1. Open the Microsoft Management Console (MMC)
-
-```text
-Start > Control Panel > Administrative Tools > Certificate Authority
-```
-
-2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
-3. Find the `Computer` template on the list, right-click on it, then select `Duplicate Template`.
-4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
-5. In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
-6. Go to the `General` tab and change `Template display name` to `RemoteDesktopAccess`. Make sure `Template name` is also `RemoteDesktopAccess`.
-7. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
-8. Next, in the `Extensions` tab select `Application Polices` and click the `Edit` button.
-9. Remove all entries from the list.
-10. Go to the `Security` tab, select `Domain Controllers` and give the group `Read` and `Enroll` permissions.
-11. Finally, create a template by clicking `OK`.
-12. Go back to the Certificate Authority window and right-click on `Certificate Templates`. Then:
-
-```text
-New > Certificate Template to Issue
-```
-Select `RemoteDesktopAccess` and click `OK`.
-
-### Update GPO to use a new certificate template
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Administrative Templates > Windows Components > Remote Desktop Services > Remote Desktop Session Host > Security
-```
-
-Right-click on `Server authentication certificate template`, `Edit`, then select `Enabled` and fill `Certificate Template Name` with `RemoteDesktopAccess`.
-
-<Figure align="left" bordered caption="RDP Certificate Template">
-  ![RDP Certificate Template](../../img/desktop-access/rdp-certificate-template.png)
-</Figure>
-
-
-### Configure server certificate auto-enrollment
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Windows Settings > Public Key Policies
-```
+</Details>
 
-Double-click on `Certificate Services Client - Auto-Enrollment`, then select `Enabled` in the `Configuration Model`.
+This guide will help you configure Teleport and your Active Directory domain controller to communicate securely with each other, register available desktops with your Teleport cluster, and provide secure access to them.
 
+## Prerequisites
 
-### Ensure your GPO is updated
+<ScopedBlock scope={["oss", "enterprise"]}>
 
-If you have not done so already, ensure your GPO is updated by opening a PowerShell prompt and running:
+- One or more hosts to run the Teleport Auth and Proxy services on.
 
-```powershell
-gpupdate.exe /force
-```
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
```

Comment:
Windows Server is not necessarily required for the desktops. Though this guide may require it if it installs AD. Any suggestions for how we might make this more clear?

Reply:
I'm not sure what you're saying here. Teleport Desktop Access requires AD, and AD needs to be run on a Windows Server OS.

Reply:
Yes, but you can have 1 Windows Server box running AD and 1000 Windows (not Windows server) boxes that you connect to via Teleport.

I don't want the docs to make it sound like Teleport can only connect to Windows Server instances.

Reply:
How about:

```suggestion
- A server or virtual machine running a Windows Server operating system, to
  provide Active Directory services.
```

Reply:
How about:

```suggestion
- A server or virtual machine running a Windows Server operating system, to
  provide Active Directory services. This will provide Desktop Access to itself
  and all other Windows desktops connected to the domain.
```

Reply:
Close, but I don't love the "will provide Desktop Access" part. AD provides lots of things, but it's Teleport that's ultimately providing access.

Reply:
It was my intention that "Teleport" be the "This" in the second sentence. I will attempt to make that clearer:

```suggestion
A server or virtual machine running a Windows Server operating system, to
provide Active Directory services. Teleport communicates with Active Directory
to sign and distribute CA certificates to Windows desktops in its domain,
so Teleport can provide secure connections to those desktops.
```

Reply:
Sorry, I know this probably feels like bikeshedding but I think technical accuracy matters. How about:

> A server or virtual machine running a Windows Server operating system.
> In this guide, we'll install Active Directory on this server in order
> to support passwordless logins with Teleport.

Reply:
Doesn't feel like bike shedding at all. I also think that technical accuracy matters, I'm a technical writer. Hopefully you weren't insinuating otherwise.
Please feel free to educate me on how my version was technically inaccurate, hopefully in another medium. In the meantime, I'm fine with your version, with one amendment to resolve one of your earlier concerns:

> Yes, but you can have 1 Windows Server box running AD and 1000 Windows (not Windows server) boxes that you connect to via Teleport.
>
>I don't want the docs to make it sound like Teleport can only connect to Windows Server instances.

```suggestion
- A server or virtual machine running a Windows Server operating system.
  In this guide, we'll install Active Directory on this server in order
  to support passwordless logins with Teleport to the Windows desktops
  in the Active Directory domain.
```

Reply:
Yep, looks good! 🚢 it!